ledger/util.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2019 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ledger
+
+import (
+	"bytes"
+
+	"github.com/spaolacci/murmur3"
+)
+
+var (
+	// DefaultLeaf is the Trie default value : hash of 0x0
+	DefaultLeaf = Hasher([]byte{0x0})
```

Comment:
Should all public symbols in this file be private?

Reply:
Yes.